axelor-stock/src/main/java/com/axelor/apps/stock/service/MassStockMoveServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.axelor.apps.stock.service;
+
+import com.axelor.apps.base.AxelorException;
+import com.axelor.apps.base.db.Company;
+import com.axelor.apps.base.db.repo.SequenceRepository;
+import com.axelor.apps.base.db.repo.TraceBackRepository;
+import com.axelor.apps.base.service.administration.SequenceService;
+import com.axelor.apps.stock.db.MassStockMove;
+import com.axelor.apps.stock.db.repo.MassStockMoveRepository;
+import com.axelor.apps.stock.exception.StockExceptionMessage;
+import com.axelor.i18n.I18n;
+import com.google.inject.Inject;
+
+public class MassStockMoveServiceImpl implements MassStockMoveService {
+
+  protected MassStockMoveRepository massStockMoveRepository;
+
+  @Inject
+  public MassStockMoveServiceImpl(MassStockMoveRepository massStockMoveRepository) {
+    this.massStockMoveRepository = massStockMoveRepository;
+  }
+
+  @Inject protected SequenceService sequenceService;
+
```

Comment:
please fix the injection: MassStockMoveRepository is not used in this service...

Also dependency must be injected with a constructor